client/views/users/editView.html

```diff
@@ -76,7 +76,7 @@
               <div class="form-group">
                 <label class="col-md-4 control-label"></label>
                 <div class="col-md-8">
-                  <input type="submit" class="btn btn-large btn-success" value="Save HMIS Profile" />
+                  <input type="submit" class="btn btn-large btn-success" value="Sign in to Home" />
                 </div>
```

Comment:
@PoojaGandhi123 This is not a login form. We are saving user details in HMIS profile here.